app/src/main/java/com/elmeradrianv/shesafe/LoginActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.elmeradrianv.shesafe;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+
+
+import com.parse.ParseUser;
+
+public class LoginActivity extends AppCompatActivity {
+    public static final String TAG="LoginActivity";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+
+        EditText etUsername=findViewById(R.id.etUsername);
```

Comment:
looks like you still have the "auto formatter" turned off, make sure to put that on!